quadrat/block-template-parts/header.html

```diff
@@ -1,8 +1,19 @@
-<!-- wp:group {"tagName":"header","layout":{"type":"flex"},"style":{"spacing":{"padding":{"bottom":"170px"}}},"className":"site-header"} -->
-<header class="wp-block-group site-header" style="padding-bottom:170px">
-	<!-- wp:site-logo /-->
-	<!-- wp:site-title /-->
-	<!-- wp:site-tagline {"style":{"typography":{"fontSize":"var(--wp--custom--font-sizes--tiny)"}}} /-->
-	<!-- wp:navigation {"__unstableLocation":"primary","__unstableSocialLinks":"social","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"right","orientation":"horizontal"}} /-->
-</header>
-<!-- /wp:group -->
+<!-- wp:group {"className":"site-header","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"bottom":"80px","top":"var(--wp--custom--gap--vertical)"}}}} -->
```

Comment:
I think the padding bottom for Quadrat is 170px, else we could just let the theme inherit the template part from the parent

Reply:
There are more differences than that.  In this change the <group> wrapping the title and tagline is a row in Quadrat & Co.  

Thanks for catching the gap size though.  I have corrected it.